### ISSUE TITLE
Status bar disappears when log view is resized to max height

### ISSUE THREAD

**Issue by andreasgeier** [camunda/camunda-modeler#2258]
__Describe the Bug__

Status bar disappears when log view is resized to max height

__Steps to Reproduce__

1. Open log view
2. Resize log view to max height

--> Status bar disappears

https://user-images.githubusercontent.com/938244/117791561-be011c80-b24a-11eb-8dad-43753569d610.mov

__Expected Behavior__

The status bar should be a static part of the UI and shall not disappear or move.

__Environment__

- OS: Mac OS 11.3.1
- Camunda Modeler Version: 4.7.0-nightly.20210407
- Installed plug-ins: cloud connect

**Comment by barmac** [comment]
It happens because of no height limit for the log. Once it reaches some height, it starts to grow to the bottom. Then, the window can be scrolled.

So to fix the issue, we need a height limit for the log.

**Comment by andreasgeier** [comment]
As part of this bug fix you could also consider ensuring that the properties panel handler does not overlap the header but stays within the tab area.

**Comment by GeethaParthasarathy** [comment]
This issue also seems to occur with DMN as well. After expanding the logs, the DMN table goes completely inaccessible. We could not view the contents of the table and scroll through them as well.

https://user-images.githubusercontent.com/85225325/124932939-c46e0380-e003-11eb-85c3-7a9cfe53baf1.mp4

**Comment by AsianCat54x** [comment]
Can I work on this issue?

**Comment by barmac** [comment]
Yes, you can. Make sure to follow [the contributing guidelines](https://github.com/camunda/camunda-modeler/blob/a61cf00db1a147d0cfad165574dc7141fb62cec1/.github/CONTRIBUTING.md).

**Comment by AsianCat54x** [comment]
Alright! Thanks! I would read and start ;)
